### PR TITLE
Added jest coverage badges

### PR DESCRIPTION
**Pull request by dashcraft**
All i did was add coverage and badges to the @artemisag/mobx-async-store repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+![Statements Covered!](./coverage/badge-statements.svg)
```
Comment (sd) [inline README.md]: You should also remove the lines in TODO for "add badges" and "publish to registries", and while you're at it, correct the end of the sentence just above that that says "for use in production use."

File: coverage/coverage-summary.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{"total": {"lines":{"total":634,"covered":523,"skipped":0,"pct":82.49},"statements":{"total":668,"covered":551,"skipped":0,"pct":82.49},"functions":{"total":185,"covered":149,"skipped":0,"pct":80.54},"branches":{"total":298,"covered":227,"skipped":0,"pct":76.17}}
+,"C:\\Users\\danie\\Documents\\workspace\\Artemis\\mobx-async-store\\spec\\ExampleApp.js": {"lines":{"total":22,"covered":22,"skipped":0,"pct":100},"functions":{"total":9,"covered":9,"skipped":0,"pct":100},"statements":{"total":22,"covered":22,"skipped":0,"pct":100},"branches":{"total":0,"covered":0,"skipped":0,"pct":100}}
```
Comment (sd) [inline coverage/coverage-summary.json]: this explicit path to your user workspace makes me think something is not correctly set up

Comment (sd) [reply coverage/coverage-summary.json]: perhaps we should be ignoring this files, and everything else in `coverage` except for the svg badges?

Comment (dashcraft) [reply coverage/coverage-summary.json]: Yup, that's it, gotta ignore everything but the badges

**Pull request merged**